### ISSUE TITLE
NVDA: Ctrl+Alt+N doesn't run a chunk because it's used to turn NVDA on and off

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#6285]
### System details

    RStudio Edition : Desktop and Server
    RStudio Version : 1.3.842
    OS Version      : Windows 10
    R Version       : 3.6.2

### Steps to reproduce the problem

With NVDA installed, open an R Markdown file and click <kbd>Ctrl+Alt+N</kbd> in order to run the next chunk.

### Describe the problem in detail

NVDA opens or closes, and the chunk isn't run.

### Describe the behavior you expected

The chunk would be run. The RStudio shortcut for running the next Sweave or Rmd chunk and the NVDA shortcut for turning it on and off is the same, <kbd>Ctrl+Alt+N</kbd>. I don't know what a solution would be. Changing it in RStudio is not ideal, since the current shortcut is familiar to many current Windows users. However, even if changing the NVDA shortcut is possible, it's a non-starter just to accommodate RStudio. Is it possible to assign two keyboard shortcuts to the same task?

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
We cannot avoid keyboard shortcut clashes, especially with screen readers which each have their own unique (and extensive) list of shortcuts. Closing this as by-design.

NVDA can be installed to not have this shortcut for starting it, the user can change it to something else, or they can change it in RStudio.

In NVDA's case, this shortcut key is simply a property of the icon created when you install it.

![screen shot of NVDA shortcut system properties](https://user-images.githubusercontent.com/10569626/75079357-516ef600-54bd-11ea-96af-1e35c92b5aac.png)
